.gitignore

```diff
@@ -22,6 +28,8 @@ __pycache__
 # build system
 /bin
 /.bin
+build
+deps
```

Comment:
I prefer disallowing such generic names throughout the repository (I found it useful for embedding thirdparty libraries). I can just make the change here.